jupyter_collaboration/app.py

```diff
@@ -57,6 +57,14 @@ class YDocExtension(ExtensionApp):
         directory.""",
     )
 
+    server_side_execution = Bool(
```

Comment:
Maybe there could also be a small note about this flag in the documentation? https://jupyterlab-realtime-collaboration.readthedocs.io/en/latest/

So it's more visible and discoverable.

Reply:
And also mentioning it only works with `jupyverse` for now